charts/seed-monitoring/charts/prometheus/templates/config.yaml

```diff
@@ -179,17 +176,23 @@ data:
       # Drop metrics that are useless and high-volume
       # Use topk(20, count by (__name__, job)({__name__=~".+"})) to find offending time-series
       - source_labels: [ __name__ ]
-        regex: container_network_tcp_usage_total
-        action: drop
-      - source_labels: [ __name__ ]
-        regex: container_tasks_state
-        action: drop
-      - source_labels: [ __name__ ]
-        regex: container_network_udp_usage_total
+        regex: ^container_({{ .Values.cadvisorAllowedMetrics | join "|" }})$
+        action: keep
+      - source_labels: [ container_name ]
+        regex: ^POD$
         action: drop
-      - source_labels: [ __name__ ]
-        regex: container_cpu_load_average_10s
+        # drop terraform pods
+      - source_labels: [ pod_name ]
+        regex: ^.+\.tf-pod.+$
```

Comment:
Why we have to drop terraform pod metrics in the Shoot cluster? This is the scrape job of the `cadvisor` for the kubelets of the Shoot and not for the kubelets of the Seed, right?

Reply:
Yes, that makes sense, but this is the job to scrape the `cadvisor` of the kubelets, which are running on the **Shoot** nodes, right? In the Shoot will not run any Gardener terraform pods, therefore there will be no such metrics to drop? Am I right?